src/responses.ts

```diff
@@ -169,3 +169,19 @@ export interface NearChunkDetails
 export interface NearChunkDetailsResponse extends NearChunkDetails {
   id: string
 }
+
+export interface GetStateResponse {
+  result: {
+    values: [
+      {
+        key: string
+        value: string
+        proof: []
+      },
+    ]
+    proof: any[]
+    block_height: number
+    block_hash: string
+  }
+  id: string
+}
```

Comment:
I think we don't usually need the `result` key here, since that should already be handled by the `send` method. You can check by logging the response from `const stateResponse = await this.send<GetStateResponse>('query', getStateParams)` and seeing if it matches the interface.
I see that `NearChunkDetails` above could have the same issue, if it does, can you please fix both?

Reply:
Sure, no problem, I already fixed it, thanks!